app/Listeners/SpeedtestCompletedListener.php

```diff
@@ -73,5 +74,19 @@ public function handle(ResultCreated $event): void
                 }
             }
         }
+
+        if ($this->notificationSettings->webhook_enabled) {
+            if ($this->notificationSettings->webhook_on_speedtest_run && count($this->notificationSettings->webhook_urls)) {
+                foreach ($this->notificationSettings->webhook_urls as $url) {
+                    Http::post($url['url'], [
+                        'result_id' => $event->result->id,
```

Comment:
Since consuming apps might require that the request body is structured in a specific way, should this body be configurable? (similar to something like https://docs.overseerr.dev/using-overseerr/notifications/webhooks#json-payload)